### DISCUSSION TITLE
Are there any logo for channels ?

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#56] [Q&A] [answered]
I can't find US m3u8 text inside of it with all channels along with logo. For an example, REWIND TV logo, ANTENNA TV, BOUNCE XL logo etc. Are there ?

**Comment by BellezaEmporium** [comment, marked as answer]
You may take a look on the [database](https://github.com/iptv-org/database) repository to see if we included logos along with the mentioned channels.

**Comment by sguinetti** [comment]
There is a channel called NHK Kishou Saigai, which warns about possible earthquakes in the country. In this case it would be a channel to inform the population and could fit as Teleservice [Not 24/7].
